src/test/groovy/org/prebid/server/functional/tests/TopicsHeaderSpec.groovy

```diff
@@ -249,24 +238,48 @@ class TopicsHeaderSpec extends BaseSpec {
-                1,
-                [PBSUtils.getRandomNumber().toString()],
-                PBSUtils.randomNumber)
-        def twoSecBrowsingTopic = SecBrowsingTopic.defaultSetBrowsingTopic(
-                2,
-                [PBSUtils.getRandomNumber().toString()],
-                PBSUtils.randomNumber)
+        def oneSecBrowsingTopic = SecBrowsingTopic.defaultSetBrowsingTopic(1)
+        def twoSecBrowsingTopic = SecBrowsingTopic.defaultSetBrowsingTopic(2)
         def headers = [(HttpUtil.SEC_BROWSING_TOPICS_HEADER): oneSecBrowsingTopic.getValidAsHeader() +
                 twoSecBrowsingTopic.getValidAsHeader()]
 
         when: "PBS processes auction request"
-        def response = defaultPbsService.sendAuctionRequestRaw(bidRequest, headers)
+        def response = prebidServerServiceWithTopicsDomain.sendAuctionRequestRaw(bidRequest, headers)
 
         then: "Bidder request should contain user.data from header"
         def bidderRequest = bidder.getBidderRequest(bidRequest.id)
         assert bidderRequest.user.data.size() == 2
-        assert bidderRequest.user.data.segment.collectMany { it -> it.id }.containsAll([oneSecBrowsingTopic.segments[0], twoSecBrowsingTopic.segments[0]].sort())
+        assert bidderRequest.user.data.name == [PRIVACY_SENDBOX_DOMAIN, PRIVACY_SENDBOX_DOMAIN]
+        assert bidderRequest.user.data.segment.collectMany { it -> it.id }
+                .containsAll([oneSecBrowsingTopic.segments[0], twoSecBrowsingTopic.segments[0]])
+
+        and: "Response should contain Observe-Browsing-Topics header"
+        assert response.headers["Observe-Browsing-Topics"] == "?1"
+    }
+
+    def "PBS should populate user.data with empty name when privacy sand box present with empty name"() {
+        given: "PBS with empty auction privacy sand box"
+        def prebidServerServiceWithTopicsDomain
+                = pbsServiceFactory.getService(["auction.privacysandbox.topicsdomain": ""])
+
+        and: "Default basic BidRequest with generic bidder"
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            user = new User()
+        }
+
+        and: "Prepare valid Sec-Browsing-Topic value header"
+        def secBrowsingTopic = SecBrowsingTopic.defaultSetBrowsingTopic()
+        def headers = [(HttpUtil.SEC_BROWSING_TOPICS_HEADER): secBrowsingTopic.getValidAsHeader()]
+
+        when: "PBS processes auction request"
+        def response = prebidServerServiceWithTopicsDomain.sendAuctionRequestRaw(bidRequest, headers)
+
+        then: "Bidder request should contain user.data from header and name with empty string"
+        def bidderRequest = bidder.getBidderRequest(bidRequest.id)
+        assert bidderRequest.user.data.size() == 1
+        assert bidderRequest.user.data[0].name == ""
+        assert bidderRequest.user.data.segment.collectMany { it -> it.id }
+                .containsAll([secBrowsingTopic.segments[0]].sort())
```

Comment:
Can be simplified:
`bidderRequest.user.data[0].segment.id == secBrowsingTopic.segments`